README.md

```diff
@@ -26,17 +26,17 @@ Starchart is not meant to be, or replace commercial registrars; nor is it meant
 - Each authorized user can create subdomains of the form: `{project|host}.{username}.example.com`. For example, the Seneca user `klee` could create: `project123.klee.example.com` or `laptop.klee.example.com`. The `*.klee.example.com` portion of the subdomain is fixed for each user (i.e., the user `klee` can't create subdoimains other than `*.klee.example.com`).
 - An SSL certificate is also generated for `*.{username}.example.com` using a [DNS Challenge](https://letsencrypt.org/docs/challenge-types/#dns-01-challenge), allowing any subdomains created for this username to be accessed securely.
 - Each domain is made available for a time-limited period (for example: "1 term" or "1 year").
-- An administration layer to handle abuse or limit users when necessary (for example, being able to remove or blacklist an entire user or subdomain)
+- An administration layer to handle abuse or limit users when necessary (for example, being able to remove or blacklist an entire user or subdomain).
 
-## Technologies
+## Technologies and components
 
-There are a number of components of the system:
-
-1. A [PowerDNS](https://www.powerdns.com/) Authoritative DNS server with REST API capable of dynamic configuration. This DNS server is replicated to a public DNS host (e.g., CIRA) and is not exposed publicly
-2. A database [backend](https://doc.powerdns.com/authoritative/backends/index.html) for PowerDNS to store its records dynamically
-3. A node.js back-end server, with the ability to connect to the DNS server's API and to authorize users via SSO
-4. A web front-end that allows authorized users to manage their own subdomains and certificates
+Here is the planned architecture and [[technologies] used(https://github.com/Seneca-CDOT/starchart/wiki/Tech-Stack)]:
```

Comment:
I assume this is not intentional:
<img width="419" alt="image" src="https://user-images.githubusercontent.com/67077705/216372836-164a3498-b325-46dc-a511-131a341c6b94.png">

Reply:
Maaaaan... I guess nvm on the brackets

Reply:
Thanks for catching it